README.md

```diff
@@ -100,9 +120,119 @@ ports:
 - **📝 POST /supply-categories** - Registra uma nova categoria de suprimentos.
 - **🔧 PUT /supply-categories/:categoryId** - Atualiza uma categoria.
 
+Claro, aqui está a documentação para esses comandos:
+
+---
+
+### Comandos Make
```

Comment:
Talvez utilizar uma tabela com a descrição dos comando deixe o readme mais limpo e mais fácil para leitura, tipo isso [aqui](https://github.com/marcieltorres/python-boilerplate-project?tab=readme-ov-file#how-to-install-run-and-test):

![image](https://github.com/SOS-RS/backend/assets/3493966/82b5f97c-f3f3-4ffc-85a3-edcdd33f1553)

Reply:
achei bem interessante o formato em tabela